packages/web/app/src/components/v2/icon.tsx

```diff
@@ -149,21 +149,6 @@ export const ArrowDownIcon: FC<IconProps> = ({ className }) => (
   </svg>
 );
 
-export const MailIcon: FC<IconProps> = ({ className, ...props }) => (
```

Comment:
Let’s keep icons from this file, we’ll add storybook after with icons stories

Reply:
I don't see why we should keep unused stuff within the code-base? What is the point of it?

Reply:
I want only keep icons, as they are already dark-mode friendly, we’ll use them in the future

Reply:
When we use them you can just pick them from the git history? I don't see the point in having commented/dead code on the `main` branch..

Reply:
Ok remove it